__tests__/app.tests.js

```diff
@@ -66,7 +66,7 @@ describe('GET /api/articles/:article_id', () => {
                 expect(article).toMatchObject({
                     author: expect.any(String),
                     title: expect.any(String),
-                    article_id: expect.any(Number),
+                    article_id: article.article_id,
```

Comment:
I like this should really be article_id : 1 that way you are absolutely sure the correct article_id coming back